teamB/JeongWooJin/app.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const { DataSource } = require('typeorm');
+
+const app = express();
+const PORT = process.env.PORT
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+appDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  })
+  .catch((err) => {
+      console.error("Error during Data Source initialization", err)
+  appDataSource.destroy()
+  })
+
+  app.use(cors());
+  app.use(morgan('dev'));
+  app.use(express.json());
+
+  app.get("/ping", (req,res) => {
+    res.status(200).json({"message" : "pong"});
+  })
+
+  app.post("/users", async (req, res, next) => {
+    const { name, email, profile_image, password } = req.body;
```

Comment:
```suggestion
    const { name, email, profileImage, password } = req.body;
```
코딩에서의 naming style에는 CamelCase 와 snake_case가 있습니다.

node.js는 camelCase를 사용하며 javacript의 또한 필수적인것은 아니지만 대체로 CamelCase를 사용하고 있습니다.

다른 코드들과 통일성을 위해 요청의 바디에 들어오는 정보들을 구조 분해 할당을 통해 변수로 넣어줄 때도 CamelCase를 사용하여 주세요!